site/docs/reference/Connectors/materialization-connectors/google-gcs-csv.md

```diff
@@ -0,0 +1,92 @@
+
+# CSV Files in Google GCS
+
+This connector materializes [delta updates](../../../concepts/materialization.md#delta-updates) of
+Flow collections into a GCS bucket in the CSV format.
+
+The delta updates are batched within Flow, converted to CSV files, and then pushed to the GCS bucket
+at a time interval that you set.
+
+[`ghcr.io/estuary/materialize-gcs-csv:dev`](https://ghcr.io/estuary/materialize-gcs-csv:dev)
+provides the latest connector image. You can also follow the link in your browser to see past image
+versions.
+
+## Prerequisites
+
+To use this connector, you'll need:
+
+* A GCS bucket to write files to. See [this
+  guide](https://cloud.google.com/storage/docs/creating-buckets) for instructions on setting up a
+  new GCS bucket.
+* A Google Cloud [service account](https://cloud.google.com/docs/authentication/getting-started)
+  with [`roles/storage.objectAdmin`](https://cloud.google.com/storage/docs/access-control/iam-roles#standard-roles)
+  for the GCS bucket created above.
+* A key file for the service account.
+
+## Configuration
+
+Use the below properties to configure the materialization, which will direct one or more of your
+Flow collections to your bucket.
+
+### Properties
+
+#### Endpoint
+
+| Property                  | Title                 | Description                                                                                                                                   | Type    | Required/Default |
+|---------------------------|-----------------------|-----------------------------------------------------------------------------------------------------------------------------------------------|---------|------------------|
+| **`/bucket`**             | Bucket                | Bucket to store materialized objects.                                                                                                         | string  | Required         |
+| **`/credentialsJson`**    | Service Account JSON  | The JSON credentials of the service account to use for authorization.                                                                         | string  | Required         |
+| **`/uploadInterval`**     | Upload Interval       | Frequency at which files will be uploaded.                                                                                                    | string  | 5m               |
+| `/prefix`                 | Prefix                | Optional prefix that will be used to store objects.                                                                                           | string  |                  |
+| `/fileSizeLimit`          | File Size Limit       | Approximate maximum size of materialized files in bytes. Defaults to 10737418240 (10 GiB) if blank.                                           | integer |                  |
```

Comment:
Maybe add here what happens if the size limit is reached?

Reply:
I added a bit to the intro section about how files are limited to a maximum configurable size, and that each Flow collection will produce many separate files.